AppServices/Validators/ValidationExtensions/ValidationExtensions.cs

```diff
@@ -40,23 +38,24 @@ public static int ToIntAt(this string value, Index index)
             return (int)char.GetNumericValue(value, indexValue);
         }
 
+        public static bool OnlyNumbers(this string numbers)
+        {
+            return numbers.All(x => x >= '0' && x <= '9');
+        }
+
         public static bool OnlyLetters(this string letters)
         {
             letters.ToLower().All(x => x >= 'a' && x <= 'z');
             string.IsNullOrWhiteSpace(letters);
             return letters.Length > 0;
-        }
-
-        public static bool OnlyNumbers(this string numbers)
-        {
-            return numbers.All(x => x >= '0' && x <= '9');
-        }
+        } 
 
         public static bool ApproveAge(this DateTime birthdate)
         {
-            var idade = DateTime.UtcNow.Year - birthdate.Year;
-            if (idade >= 18) return true;
-            return false;
+            var idade = new DateTime(DateTime.Now.Subtract(birthdate).Ticks).Year - 1;
+            return idade >= 18
+                ? true
+                : false;
```

Comment:
Não programamos em português
```suggestion
            var age = new DateTime(DateTime.Now.Subtract(birthdate).Ticks).Year - 1;
            return age >= 18;
```